step.sh

```diff
@@ -1,12 +1,26 @@
 #!/bin/bash
 set -ex
 
-if [[ "${ruby_version}" ]]; then
-	asdf install ruby "${ruby_version}"
-	asdf global ruby "${ruby_version}"
+# Check if the Gemfile already exists
+if [[ -f "${gemfile_path}" ]]; then
+  echo "Gemfile already exists at ${gemfile_path}"
+
+  # Install gems using the existing Gemfile
+  BUNDLE_GEMFILE="${gemfile_path}" bundle install
```

Comment:
Can we call `bundle install` this after if/else check?
```suggestion
  BUNDLE_GEMFILE="${gemfile_path}" 
```

Just to avoid duplicating the command inside if/else when the command should be run either way.